Analysis/src/ConstraintGenerator.cpp

```diff
@@ -32,6 +32,7 @@ LUAU_FASTINT(LuauCheckRecursionLimit)
 LUAU_FASTFLAG(DebugLuauLogSolverToJson)
 LUAU_FASTFLAG(DebugLuauMagicTypes)
 LUAU_FASTFLAG(LuauPreserveUnionIntersectionNodeForLeadingTokenSingleType)
+LUAU_FASTFLAG(LuauPropagateExpectedTypesForCalls)
```

Comment:
```suggestion
LUAU_FASTFLAGVARIABLE(LuauPropagateExpectedTypesForCalls)
```

One file must contain the definition of this flag, and this is the one where it belongs.